### PR TITLE
350 fix resource and service inactivation

### PR DESCRIPTION
**Pull request by jjfreund**

### CODE REVIEW

File: lib/tasks/algolia.rake [hunk 1]
```diff
@@ -3,8 +3,8 @@
 namespace :algolia do
   task reindex: :environment do
     print "Reindexing resource/service index... "
-    Resource.reindex!
-    Service.reindex!
+    Resource.where(status: :approved).reindex
```
Comment (lgarofalo) [inline lib/tasks/algolia.rake]: With the `.where`, can this become null? (looking at testerrors)

Comment (richardxia) [reply lib/tasks/algolia.rake]: Looking at the Travis errors, it looks like it's actually erroring on the `remove_from_index!` calls in the other lines of the diff.

Comment (lgarofalo) [reply lib/tasks/algolia.rake]: Ah ok, I'll take a deeper look.

**Comment by lgarofalo** [comment]
@jjfreund good to merge?

**Pull request merged**